documentation/design/user-stories/shared-infra.adoc

```diff
@@ -115,32 +80,28 @@ metadata:
   name: myqueue
   namespace: myapp
 spec:
+  anycast: {}
+  multicast: {}
   queue: {}
+  topic: {}
+  subscription:
+    # References a MessagingAddress named mytopic with type topic.
+    topic: mytopic
+
+  # A system-wide address available on all brokers
+  deadLetter: {}
+status:
+  phase: Pe
 ```
 
+---
+
 === Ability to configure DLQ, expiry, forwarders etc. per address
 
-Define a special system queue that gets created on all brokers:
+* Queues that should be configured with a DLQ or ExpiryQueue should reference an address with type `deadLetter`.
+* Forwarders maybe configured for the queue (`in` and `out`), subscription (`out`) or deadLetter(`out`)) types.
+* If the referenced DLQ does not exist, the status section of the address will provide an error message.
```

Comment:
Just an observation, but if you forward an expired messages, won't it spontaneously expire again at its new destination.

Reply:
Good point, we should note that in the docs then.

Reply:
well, can queues be configured so that they effectively ignore expiry?  You'd want that generally for DLQs / expiry queues

Reply:
Answering my own original question, no.  When artemis expires a message., internally it actually creates a copy of the message for the expiry queue, and set the copy's expiry to infinity.  The message's original expiry is set in a custom application property.  Not an AMQP compliant approach, but understandable one for an multi protocol broker.

org.apache.activemq.artemis.core.server.impl.QueueImpl#makeCopy(org.apache.activemq.artemis.core.server.MessageReference, boolean, boolean)